src/chttpd/src/chttpd.erl

```diff
@@ -326,7 +326,7 @@ handle_request_int(MochiReq) ->
          || Part <- string:tokens(Path, "/")
         ],
         requested_path_parts = [
-            ?l2b(unquote(Part))
+            ?l2b(chttpd:unquote(Part))
```

Comment:
This doesn't need to change unless we want to mock chttpd:unquote/1 we can keep using the local version